features/admin.validation.v1/pages/validation-config-edit.tsx

```diff
@@ -1038,19 +1020,13 @@ export const ValidationConfigEditPage: FunctionComponent<MyAccountSettingsEditPa
                         max={
                             currentValues.maxLength
                                 ? currentValues.maxLength
-                                : ValidationConfigConstants
-                                    .VALIDATION_CONFIGURATION_FORM_FIELD_CONSTRAINTS
-                                    .PASSWORD_MAX_VALUE
+                                : maxPasswordLenghtLimit
                         }
                         width={ 2 }
                         required
                         hidden={ false }
                         placeholder={ "min" }
-                        maxLength={
-                            ValidationConfigConstants
-                                .VALIDATION_CONFIGURATION_FORM_FIELD_CONSTRAINTS
-                                .PASSWORD_MAX_LENGTH
-                        }
+                        maxLength={ maxPasswordLenghtLimitLength }
```

Comment:
Do we need maxlength for a number input, given that min and max values are already provided?

Reply:
Yes we don't need this validation but this is comes from `FieldInputPropsInterface` and there are multiple occurrences if we are going to modify it.

Reply:
Ack